client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -494,29 +508,34 @@ void main(void)
 		// calculate fragment color from components:
 		col.rgb =
 				adjusted_night_ratio * col.rgb + // artificial light
-				(1.0 - adjusted_night_ratio) * ( // natural light
+				sunTint * (1.0 - adjusted_night_ratio) * ( // natural light
 						col.rgb * (1.0 - shadow_int * (1.0 - shadow_color) * (1.0 - shadow_tint)) +  // filtered texture color
 						dayLight * shadow_color * shadow_int);                 // reflected filtered sunlight/moonlight
 
 
 		vec3 reflect_ray = -normalize(v_LightDirection - fNormal * dot(v_LightDirection, fNormal) * 2.0);
 
-		vec3 viewVec = normalize(worldPosition + cameraOffset - cameraPosition);
-
 		// Water reflections
-#if (defined(MATERIAL_WAVING_LIQUID) && defined(ENABLE_WATER_REFLECTIONS) && ENABLE_WAVING_WATER)
+#if (defined(MATERIAL_WAVING_LIQUID) && defined(ENABLE_WATER_REFLECTIONS))
+
+#if !ENABLE_WAVING_WATER
+#define WATER_WAVE_HEIGHT 0.5
```

Comment:
We could also still use the values from settings and document that they are for waving water and/or reflection.